### PR TITLE
Release 2.5.5 with JDK9 and juc.Flow support!

### PR DESCRIPTION
**Pull request by ktoso**
Please merge https://github.com/akka/akka.github.com/pull/463 before this one, as we point to it in this news item

### CODE REVIEW

File: blog/_posts/2017-09-28-akka-2.5.5-released.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+# ANNOUNCE: Akka 2.5.5
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. 
+Other than the usual hardening and maintanance work, during this period we made a lot of progress on multiple very important new features which we’re very proud to release today.
+
+The most important features are:
+
+* Support for Java 9 across the board
```
Comment (patriknw) [inline blog/_posts/2017-09-28-akka-2.5.5-released.md]: stupid question, do we have to say that we still support Java 8?

File: blog/_posts/2017-09-28-akka-2.5.5-released.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+short: We are happy to announce a new patch release of Akka 2.5
+category: news
+tags: [releases]
+---
+# ANNOUNCE: Akka 2.5.5
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. 
+Other than the usual hardening and maintanance work, during this period we made a lot of progress on multiple very important new features which we’re very proud to release today.
```
Comment (johanandren) [inline blog/_posts/2017-09-28-akka-2.5.5-released.md]: maintenance

File: blog/_posts/2017-09-28-akka-2.5.5-released.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    * Cluster Sharding [#23698](https://github.com/akka/akka/issues/23698)
+    * Distributed Data [#23664](https://github.com/akka/akka/issues/23664) 
+    * and the Receptionist [#23634](https://github.com/akka/akka/issues/23634)
+* First release of [Multi-DC Clustering](https://doc.akka.io/docs/akka/current/scala/cluster-dc.html) (preview)
+
+A total of 99 issues were closed since 2.5.4. The complete list can be found on the [2.5.5](https://github.com/akka/akka/milestone/118?closed=1) milestone on github.
+
+## Request for feedback of Akka Typed APIs
+
+Please note that all the Akka Typed APIs (persistence, cluster, ...) remain "[ApiMayChange](https://doc.akka.io/docs/akka/2.5.4/scala/common/may-change.html)",
```
Comment (johanandren) [inline blog/_posts/2017-09-28-akka-2.5.5-released.md]: Hard link to previous version of docs?

File: blog/_posts/2017-09-28-akka-2.5.5-released.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+A total of 99 issues were closed since 2.5.4. The complete list can be found on the [2.5.5](https://github.com/akka/akka/milestone/118?closed=1) milestone on github.
+
+## Request for feedback of Akka Typed APIs
+
+Please note that all the Akka Typed APIs (persistence, cluster, ...) remain "[ApiMayChange](https://doc.akka.io/docs/akka/2.5.4/scala/common/may-change.html)", 
+as we continue to gather feedback and finalize those APIs over the coming months. We’d like to encourage you to try them out and provide feedback in case you 
+encounter some missing or confusing functionality. 
+
+These new APIs do not yet have a full-blown documentation, however we will introduce all the APIs in a fresh blog post series, 
+begining with the [Typed Akka Cluster API post](http://akka.io/blog/2017/09/28/typed-cluster) published *right now*, where
```
Comment (johanandren) [inline blog/_posts/2017-09-28-akka-2.5.5-released.md]: https

File: blog/_posts/2017-09-28-akka-2.5.5-released.md (hunk 1 above)
Comment (ktoso) [reply blog/_posts/2017-09-28-akka-2.5.5-released.md]: Good point, let's spell it out

File: blog/_posts/2017-09-28-akka-2.5.5-released.md (hunk 2 above)
Comment (ktoso) [reply blog/_posts/2017-09-28-akka-2.5.5-released.md]: that word is my nemesis :O

File: blog/_posts/2017-09-28-akka-2.5.5-released.md (hunk 4 above)
Comment (ktoso) [reply blog/_posts/2017-09-28-akka-2.5.5-released.md]: thanks

File: blog/_posts/2017-09-28-akka-2.5.5-released.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+# ANNOUNCE: Akka 2.5.5
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. 
+Other than the usual hardening and maintenance work, during this period we made a lot of progress on multiple very important new features which we’re very proud to release today.
+
+The most important features are:
+
+* Support for Java 9 across the board (while maintaining Java 8 support)
```
Comment (ktoso) [inline blog/_posts/2017-09-28-akka-2.5.5-released.md]: called out java 8 compat here

**Comment by ktoso** [comment]
Thanks for review, fixed

**Pull request merged**